wordpress-mobile/WordPress-Android
# Adds support for adding authorization headers to glide requests

Fixes #7823

Adds support for adding authorization headers to glide requests. Since we don't use glide for loading any private resources yet(it'll be part of another PR), we can just test the changes haven't broken anything. 


To test 
1. Open the app
2. Click on the Log in button
3. Enter an email address
4. Click on the Next button
5. Make sure the avatar image loads properly
---
1. Open the app
2. Click on the Log in button
3. Click on the "login by entering your site address"
4. Enter a site address
5. Make sure the blavatar image loads properly
